schemas/0.1/schemas/json_schema/image.schema

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Would be good to discuss _not_ including these in this repo.

Reply:
I almost commented on whether `ngff_schemas` could be pip installable but since you've opened the door... ;)

Reply:
Sorry, yes - the intention was not to include them, but just put them there to get this working.
Do we just add a download_schema command somewhere that is run on the fly if the schemas are missing, and can also be run when we package for release?

Reply:
I don't think `ngff_schemas` should be pip installable, unless we make `ngff` pip installable since it would be nice to have them in the same repo (update the spec at the same time as the schemas).

Reply:
Making ngff pip-installable at https://github.com/ome/ngff/pull/77 and using that below in this PR...